packages/material-ui/src/Backdrop/Backdrop.js

```diff
@@ -47,11 +47,15 @@ const Backdrop = React.forwardRef(function Backdrop(props, ref) {
 });
 
 Backdrop.propTypes = {
+  /**
```

Comment:
Should I add a comment here to notify people modifying the source that they should edit the definitions instead?